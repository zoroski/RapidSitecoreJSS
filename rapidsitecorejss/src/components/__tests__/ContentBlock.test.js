import React from 'react';
import ContentBlock from '../ContentBlock/index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
      const tree = renderer
        .create( < ContentBlock fields = {
            {
              heading: {
                value: 'Take note: Even the Reserve Bank of Australia needs sub-editors',
              },
              content: {
                value: 'Central banks have a lot to do, so itâ€™s no surprise things get overlooked once in a while.',
              },
            }
          }
          />)
          .toJSON(); expect(tree).toMatchSnapshot();
        });