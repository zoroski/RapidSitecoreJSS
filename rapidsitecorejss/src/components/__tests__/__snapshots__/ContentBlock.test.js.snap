// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <h2
    className="display-4"
  >
    Take note: Even the Reserve Bank of Australia needs sub-editors
  </h2>,
  <div
    className="contentDescription"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Central banks have a lot to do, so itâ€™s no surprise things get overlooked once in a while.",
      }
    }
  />,
]
`;
